packages/miniflare/src/workers/queues/schemas.ts

```diff
@@ -26,7 +26,7 @@ export const QueueConsumerOptionsSchema = /* @__PURE__ */ z.object({
 	// https://developers.cloudflare.com/queues/platform/limits/
 	maxBatchSize: z.number().min(0).max(100).optional(),
 	maxBatchTimeout: z.number().min(0).max(30).optional(), // seconds
-	maxRetires: z.number().min(0).max(100).optional(),
+	maxRetries: z.number().min(0).max(100).optional(),
```

Comment:
I don't think we can actually remove this, since it's part of the Miniflare public API surface. Instead, could you add a new `maxRetries` property and make it fall back to `maxRetires` if not defined?

Reply:
Oh, didn't realize that. Yeah I can do this!